utils/sec_utilities/edgar_ingestion.py

```diff
@@ -0,0 +1,96 @@
+import yaml
```

Comment:
Can we please move this to a utils within the yoda  kit repo? This isn't a multi kit util.